srml/contract/src/lib.rs

```diff
@@ -261,18 +283,27 @@ where
 		});
 
 		let mut buf = Vec::new();
-		buf.extend_from_slice(account_id.as_ref());
+		buf.extend_from_slice(b"contract_seed");
```

Comment:
I'm not sure what this contract_seed means

Reply:
that is a constant prefix for this module (length and content may not be optimal) in order to avoid other similar keyspace generator (that simple counter implementation) to create conflicting keyspace.